packages/ui/src/theme/tokens/borderWidths.ts

```diff
@@ -0,0 +1,13 @@
+import { DesignToken } from './types/designToken';
+
+export interface BorderWidths {
```

Comment:
Originally I had all the types in the types folder, but felt weird to have them so far from being used. Is this ok or should they go into a types folder?